cache/clock_cache.cc

```diff
@@ -43,69 +45,67 @@ ClockHandleTable::~ClockHandleTable() {
 
 ClockHandle* ClockHandleTable::Lookup(const Slice& key, uint32_t hash) {
   uint32_t probe = 0;
-  int slot = FindElement(key, hash, probe);
-  return (slot == -1) ? nullptr : &array_[slot];
+  ClockHandle* e = FindSlot(
+      key,
+      [&](ClockHandle* h) {
+        if (h->TryInternalRef()) {
+          if (h->IsElement() && h->Matches(key, hash)) {
```

Comment:
I know it wasn't introduced in this PR, but ```IsElement()``` is not very clear. Maybe something like ```IsValid()``` would have been more appropriate.

Reply:
A handle can be 3 different things: a KV element, a tombstone or an empty slot. Tombstones and empty slots are not really invalid.

Maybe ``IsOccupied()``? I don't love this term either because I think of tombstones as occupying slots too.